### ISSUE TITLE
Project aims

### ISSUE THREAD

**Issue by mauicv** [mauicv/pygrl#10]
Currently `pyg` will successfully run NEAT even with some minor errors #5. The aim is to build pyg so that:
1. NEAT is easy
    - See [this](http://nn.cs.utexas.edu/downloads/papers/stanley.ec02.pdf)
2. Reinforce-ES is implementable
    - See [this](https://openai.com/blog/evolution-strategies/) and [this](https://blog.otoro.net/2017/10/29/visual-evolution-strategies/)
3. Simple to serve as a backend to a realtime mixed human selection system.
    - Idea is that using a very simple baseline reward function in a given environment we have a human select best stratigies. We both use the human data as a fitness function and train a critic network that adjusts the baseline reward function. 
    - As well as this it would be useful to be able to use policy gradient methods to fine tune evolved solutions. To do this we need to be able to convert between out Model some other ML frame work model such as keras...
